[SOAR 1789] Bug fixes and stability improvements for Block Host with FortiGate workflows (#266)
rapid7/insightconnect-workflows: workflows/Block_Host_with_Fortinet_Firewall_from_Microsoft_Teams/help.md

|Plugin|Version|Count|
|----|----|--------|
|HTML|1.2.1|1|
|Fortinet FortiGate|4.0.3|4|
|Microsoft Teams|2.0.5|9|

## Troubleshooting

_There is no troubleshooting information at this time_

# Version History

> * 1.1.0 - Update workflow to handle block requests for address that already exist | Update workflow to continue on failure and send messages indicating that something went wrong

Reviewer: ```suggestion
* 1.1.0 - Update workflow to handle block requests for address that already exists | Update workflow to send Teams message when a job failure has occurred
```
